src/app/content/practiceQuestions/components/LinkToSection.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useSelector } from 'react-redux';
+import styled, { css } from 'styled-components/macro';
+import { linkColor } from '../../../components/Typography';
+import theme from '../../../theme';
+import * as contentSelectors from '../../selectors';
+import { LinkedArchiveTreeSection } from '../../types';
+import { getBookPageUrlAndParams } from '../../utils/urlUtils';
+
+// tslint:disable-next-line: variable-name
+export const StyledContentLink = styled.a`
+  display: block;
+  font-size: 1.4rem;
+  color: #929292;
+  margin-top: 2.5rem;
+  text-decoration: none;
+
+  > span {
+    color: ${linkColor};
+
+    &::before {
+      content: " ";
+    }
+  }
+
+  ${theme.breakpoints.mobile(css`
+    margin: 1.2rem;
+  `)}
+`;
+
+interface LinkToSectionProps {
+  section: LinkedArchiveTreeSection | null;
+}
+
+// tslint:disable-next-line: variable-name
+const LinkToSection = ({ section }: LinkToSectionProps) => {
+  const book = useSelector(contentSelectors.book);
+
+  const linkToTheSection = React.useMemo(() => {
+    return book && section ? getBookPageUrlAndParams(book, section).url : null;
+  }, [book, section]);
+
+  if (!section || !linkToTheSection) { return null; }
+
+  return <StyledContentLink href={linkToTheSection} target='_blank' data-analytics-label='Go to link' >
```

Comment:
why aren't you using the actual `ContentLink` component?

Reply:
We want to open this link in a new tab and `ContentLink` component doesn't handle that - I think it is because it calls `e.preventDefault` `onClick`

Reply:
but it has all this logic about the url handling already, if necessary we could add a prop for target _blank